contracts/deploy/sonic/003_sonic_staking_strategy.js

```diff
@@ -62,6 +63,18 @@ module.exports = deployOnSonic(
     );
     console.log("Initialized SonicStakingStrategy proxy and implementation");
 
+    // Deploy a new Wrapped Origin Sonic contract
+
+    const cOSonicProxy = await ethers.getContract("OSonicProxy");
+    const cWOSonicProxy = await ethers.getContract("WOSonicProxy");
+
+    const dWOSonic = await deployWithConfirmation("WOSonic", [
+      cOSonicProxy.address, // Base token
+      "Wrapped OS", // Token Name
```

Comment:
At some point we should probably override the constructor so it doesn't accept the Token Name & Symbol just to avoid confusion